rclpy/rclpy/client.py

```diff
@@ -50,7 +50,7 @@ def run(self):
 class Client:
     def __init__(
             self, node_handle, client_handle, client_pointer,
-            srv_type, srv_name, qos_profile):
+            srv_type, srv_name, qos_profile, callback_group):
```

Comment:
Maybe move the default for the callback group from the `create_client` function here?

Same for other entities.

Reply:
I think the node needs to own the default callback group so that callbacks can be mutually exclusive by default.